src/actions/ServiceGraphDataActions.ts

```diff
@@ -42,7 +42,8 @@ const decorateGraphData = (graphData: any) => {
       isGroup: undefined,
       isRoot: undefined,
       isUnused: undefined,
-      hasMissingSidecars: undefined
+      hasMissingSidecars: undefined,
+      isOutsideNamespace: undefined
```

Comment:
You need to rebase to fix conflicts (e.g. 'hasMissingSidecar's is now 'hasMissingSC').  With respect to the server PR, 'isOutsideNamespace' is now just 'isOutside'